Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -179,43 +169,34 @@ public bool GroundedCheck()
 	{
 		_isOnGround = false;
 
-		// spherecast down to detect when we're on the ground
-		RaycastHit hit;
-		Physics.SphereCast( transform.position + Vector3.up,
-		                    _jumpCheckRadius,
-		                    -Vector3.up * _jumpCheckDistance,
-		                    out hit,
-		                    _jumpCheckDistance,
-		                    _jumpLayer );
-		_isOnGround = ( hit.transform &&
-		                Vector3.Dot( hit.normal, Vector3.up ) > _minJumpDot );
-
-		if ( _isOnGround &&
-		     !_jumpCheck &&
-		     !IsInState( ActorStates.Climbing ) )
+		if ( !_jumpCheck )
```

Comment:
`_jumpCheck` is a poorly named variable. It's actually the late jump delay. I'll rename it to make things more clear.